File: Beecrowd-Problems/Python/done!/problem1011.py
class Sphere:

    def __init__(self, r):
        pi = 3.14159
        self.radius = r
        self.volume = (4/3) * pi * r * r * r
    
r = float(input())
sp = Sphere(r)
print("VOLUME = {:.3f}".format(sp.volume))

File: Beecrowd-Problems/C#/done!/problem1011.cs
using System;
using System.Globalization;

namespace Program{

    class Program{
        static void Main(string[] args){
            CultureInfo.CurrentCulture = CultureInfo.InvariantCulture;
            CultureInfo.CurrentUICulture = CultureInfo.InvariantCulture;
            double pi = 3.14159;
            double radius;
            bool isParseSuccessful = double.TryParse(Console.ReadLine(), out radius);

            if (isParseSuccessful) {
                double volume = (4.0 / 3) * pi * radius * radius * radius;
                Console.WriteLine($"VOLUME = {volume:F3}");
            }
            else {
                Console.WriteLine("Error while parsing input.");
            }
        }
    }
}